### ISSUE TITLE
Relax the ordering of table content elements?

### ISSUE THREAD

**Issue by chaals** [w3c/html#439]
[Browsers handle most table elements by semantics](http://chaals.github.io/testcases/table-section-row-ordering.html) regardless of where they appear in the table - unless you have multiple `thead`, `tfoot` or `caption` elements, as per #438. This matches the parsing section.

Relaxing the content model brings an issue of keyboard navigation. If we assume that `tfoot` elements will always be relegated to the bottom of the table, as per https://github.com/whatwg/html/issues/352 then we might want to keep the order of `caption?` `colgroup*` `thead?` `{tbody,tr}*` `tfoot?`. But table navigation for things that rely on keyboards is already more complex than that, and trying to force the least effective navigation model on browsers seems a bad reason to impose restrictions.

We might also want to keep the restriction because we like telling authors how they should be putting together their content. I'm opposed, but could live with that.

**Comment by chaals** [comment]
Leaving the content model as is clarifies what is conformant, and works. What to do when authors don't follow the instructions is not part of this issue. Closing as "this doesn't need a change".
